docker/images/pinot/Dockerfile

```diff
@@ -71,7 +71,7 @@ ENV JAVA_OPTS="-Xms4G -Xmx4G -Dpinot.admin.system.exit=false"
 VOLUME ["${PINOT_HOME}/configs", "${PINOT_HOME}/data"]
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends vim less wget curl git python && \
+    apt-get install -y --no-install-recommends vim less wget curl git python sysstat procps && \
```

Comment:
@xiangfu0 ^ this is the actual runtime image correct? 

checking there's one more `apt-get install` above in line 35 which I think that is for the image running the maven build